packages/core/logger/src/Logger.js

```diff
@@ -140,15 +140,13 @@ export class PluginLogger {
 }
 
 let consolePatched = false;
+export const INTERNAL_ORIGINAL_CONSOLE = {...console};
```

Comment:
Can this just be a direct reference to the original console?

Reply:
we overwrite the console object values with our logger functions so I don't think that would actually do anything?

Reply:
I didn't know we did this. Why override the individual functions?

Reply:
Probably because there's more to console as just the logger functions? Not sure, I think Devon wrote patchConsole